LaTeX/LaTeX.sublime-syntax

```diff
@@ -198,6 +207,123 @@ contexts:
     - include: paragraph-pop
     - include: else-pop
 
+  xparse-argspec:
+    - match: (?=\{)
+      set:
+        # deliberately using an anonymous scope here. we need two match the `{`
+        # in two stages, so we can apply the `clear_scopes`, and still be able
+        # to push `xparse-argspec` onto the stack without messing up scoping.
+        - clear_scopes: 1
+        - match: '\{'
+          scope: punctuation.definition.group.brace.begin.latex
+          set: xparse-argspec-content
+
+    - include: paragraph-pop
+    - include: else-pop
+
+  xparse-argspec-content:
+    - clear_scopes: 1
+    - meta_scope: meta.function.parameters.latex
+    - include: brace-group-end
+    - include: macro-braces
+    - include: xparse-single-argspec
+    # not actually allowed, but included here to future-proof highlighting
+    - include: general-constants
+    - include: general-commands
+    - include: paragraph-pop
```

Comment:
`paragraph-pop` doesn't appear in other comparable contexts. Is it really needed?

```suggestion
```

Reply:
it is needed in the sense that it will improve the highlighting while you are typing, if the closing brace is not there yet.